README.md

```diff
@@ -1,88 +1,135 @@
+import.
 
-Once you've added `crds-styles` to `package.json` you need to import the stylesheet in your application's main '.scss' file using the following convention. You can optionally override any of the customizable Bootstrap variables prior to import...
+```scss
+@import '~crds-styles/assets/stylesheets/bootstrap';
+```
+
+_Note: The tilde character (`~`) assumes use of the
+[node-sass](https://github.com/sass/node-sass) library.
+
+---
 
-    @import '~crds-styles/assets/stylesheets/bootstrap';
+#### A note on local development
 
-## SVGs
+When making changes to crds-styles locally and in wanting them to be reflected
+in your project, you should follow the NPM/package.json approach for importing
+styles.
 
-Any project consuming `crds-styles` will need to update their build process in order to access SVG files. For information on adding new icons to `crds-styles` please refer to [the documentation](https://github.com/crdschurch/crds-styles/blob/development/src/assets/svgs/README.md).
+In this case you'll want to link to a local copy of crds-styles. Assuming the
+`crds-styles` repo resides in the same local directory as your project, you can
+do the following to setup the symlink.
 
-### Webpack build
-If using webpack, the `copy-webpack-plugin` can be used to move the svg files to the distribution folder
-```javascript
-plugins: [
-    new CopyWebpackPlugin([
-      {
-        context: './node_modules/crds-styles/assets/svgs/',
-        from: '*.svg',
-        to: 'assets'
-      }
-    ])
-  ]
+```text
+$ cd path/to/your/project-directory
+$ rm -rf node_modules/crds-styles
+$ npm link ../crds-styles
 ```
-See `crds-styleguide` [`webpack.common.js`](https://github.com/crdschurch/crds-styleguide/blob/development/config/webpack.common.js) for a full example
 
-### Webpack Dev Server
-If using `webpack-dev-server` as a development tool, the implementation will need to be modifed to support `content-base` which will serve static assets from the given directory as well as trigger a build in order to have `CopyWebpackPlugin` triggered to move SVGs
-```json
-"scripts": {
-    "serve": "npm run start",
-    "start": "npm run build-dev && webpack-dev-server --inline --open --progress --port 4200 --content-base dist/",
-    "build-dev": "rimraf dist && webpack --config config/webpack.dev.js --progress --profile --bail",
-}
+SVGs
+--------
+
+Any project consuming `crds-styles` will need to update their build process in
+order to access SVG files. For information on adding new icons to `crds-styles`
+please refer to [the
+documentation](https://github.com/crdschurch/crds-styles/blob/development/src/assets/svgs/README.md).
+
+### Angular CLI Build
+
+Angular CLI ships with the ability to copy assets into a project. To include the
+icons (or any other assets), drop the necessary files in `src/assets`. Then make
+sure your `.angular-cli.json` file copies the assets directory:
+
+```js
+{
+  "apps": [{
+    "assets": [
+      "assets"
+    ]
+  }
+]
```

Comment:
This approach is not what I would've expected. Is there no way to copy files directly from `./node_modules/crds-styles/assets/` into our `dist/assets/` directory?

Reply:
Yeah, I thought this was simpler, but I agree—there's less room for error with your example.